packages/core/strapi/lib/services/webhook-runner.js

```diff
@@ -32,14 +32,18 @@ class WebhookRunner {
 
     this.queue = new WorkerQueue({ logger, concurrency: 5 });
     this.queue.subscribe(this.executeListener.bind(this));
+
```

Comment:
Please avoid refactor the webhook system if not necessary. It could benefit from a subscribe function but that's not the goal of this PR and it would require more testing if you did.

Reply:
I understand that we want to reduce the risk by changing as little as possible, I'll update 👍 

>that's not the goal of this PR

Does this mean I could potentially reopen it as another dedicated PR in the future?

Reply:
Yes of course 👍  you can if you have bandwidth for sure with added tests etc :)